src/react-app-env.d.ts

```diff
@@ -1 +1,6 @@
 /// <reference types="react-scripts" />
+
```

Comment:
I am unsure what this does, any suggestions for resources/googling?

Reply:
I'm unsure too. I added this after receiving an error and got from this resource: https://github.com/facebook/create-react-app/issues/10109

Reply:
Cool, thank you!